pyanaconda/payload/livepayload.py

```diff
@@ -143,6 +144,7 @@ def install(self):
         # file system boundaries
         args = ["-pogAXtlHrDx", "--exclude", "/dev/", "--exclude", "/proc/",
                 "--exclude", "/sys/", "--exclude", "/run/", "--exclude", "/boot/*rescue*",
+                "--exclude", "/boot/loader/", "--exclude", "/boot/efi/loader/",
```

Comment:
Will this work if `inst.noblscfg`  is used?

Reply:
@jkonecny12 yes, because with `inst.noblscfg` the `/boot/loader` directory is not used.

Reply:
OK, thanks for answer!